plugins/woocommerce-admin/client/marketplace/components/header/header.tsx

```diff
@@ -28,10 +28,10 @@ export default function Header( props: HeaderProps ) {
 				selectedTab={ selectedTab }
 				setSelectedTab={ setSelectedTab }
 			/>
+			<Search />
```

Comment:
Is this how you fixed the tab order of the search and user menu? That's such a great solution! The CSS Grid areas being named really helps make this easy to do. Nice.

Reply:
Yes, it's all about the order of HTML elements in the page.

Reply:
Exactly. Grids are magical. No mucking around with `tabindex="..."` (which usually cause more trouble than they're worth), just _put the items on the page in the right semantic order_, and then _use CSS to position them_.

Positioning is part of the design, therefore a stylesheet is where it belongs; order is part of the content, therefore HTML is where it belongs. Grids (and to a lesser extent, flexboxen) allow this, and that's great.